src/walletdb.cpp

```diff
@@ -1047,6 +1048,9 @@ bool CWalletDB::WriteZerocoinMint(const CZerocoinMint& zerocoinMint)
     ss << zerocoinMint.GetValue();
     uint256 hash = Hash(ss.begin(), ss.end());
 
+    LogPrintf("pub: %s\nserial: %s\nrand: %s\ncrypted: %b\n", zerocoinMint.GetValue().ToString(16), zerocoinMint
```

Comment:
suggest making this a `LogPrint("zero", ...);` instead of a `LogPrintf()` to reduce log spam.